x/foundation/client/cli/tx.go

```diff
@@ -549,6 +558,54 @@ func NewTxCmdExec() *cobra.Command {
 	return cmd
 }
 
+func NewTxCmdUpdateCensorship() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-censorship [authority] [msg-type-url] [new-authority]",
+		Args:  cobra.ExactArgs(3),
+		Short: "Update censorship over a message",
+		Long: `Update censorship over a message
+
+Parameters:
+    authority: the current authority of the censorship
+    msg-type-url: the message type url of the censorship
+    new-authority: a new authority of the censorship
+        CENSORSHIP_AUTHORITY_UNSPECIFIED: no authority, which means removing the censorship
+        CENSORSHIP_AUTHORITY_GOVERNANCE: x/gov
+        CENSORSHIP_AUTHORITY_FOUNDATION: x/foundation
```

Comment:
It's too wrong to type it. How about using enum number?

Reply:
In my opinion, It would be better to use the same style as those of TxCmdVote() for consistency.
https://github.com/Finschia/finschia-sdk/blob/main/x/foundation/client/cli/tx.go#L451C37-L488

Reply:
For the consistency, it should remain as is. However, I want to add cli breaking change to `TxCmdVote()` later. What do you think? @jaeseung-bae